deployment/docker-compose.override.yml

```diff
@@ -5,9 +5,9 @@ version: '3.4'
 x-defaults: &defaults
   restart: "no"
   volumes:
-    - ~/services-dev/state:/state
+    - ~/state:/state
```

Comment:
I think we can use
```
- ./state:/state
- ../:/services
```
to make this work regardless of where it is installed and to make sure the `state` directory already exists. It might be considered a bad thing to store the state inside the git clone, though.

Reply:
Good points, I changed it.